malaria24/ona/models.py

```diff
@@ -292,11 +310,20 @@ def get_digest_email_data(self, province, facility_code):
                     .first().create_date_time.strftime(
                         "%d %B %Y"
                     )
+                now_start_day = int(start_date.split(' ', 1)[0])
+                if now_start_day < min_start_day:
+                    min_start_day = now_start_day
+                    start_week_day = start_date
                 end_date = district_cases \
                     .last().create_date_time.strftime(
                         "%d %B %Y"
                     )
-                week = "{0} to {1}".format(start_date, end_date)
+                now_end_day = int(end_date.split(' ', 1)[0])
+                if now_end_day > max_start_day:
+                    max_start_day = now_end_day
+                    end_week_day = end_date
+                week = "{0} to {1}".format(start_week_day, end_week_day)
```

Comment:
I think we have the same problem here with the formatting, and also that this will fix the per-district date range (on line 333), but not the date range across all districts (on line 362)